torchci/pages/flakytest.tsx

```diff
@@ -6,7 +6,7 @@ const fetcher = (url: string) => fetch(url).then((res) => res.json());
 
 export default function Page() {
   const router = useRouter();
-  const name = (router.query.name || "%") as string;
+  const name = (router.query.name || "test_ddp_uneven_inputs") as string;
```

Comment:
this seems like an odd default…

Reply:
I just took one that definitely worked--another thing I could do instead of an actual example would be "test_case_exact_name_example"

Reply:
okay seems fine for now. Seems like the right thing to do here is show a null state though.